composer.json

```diff
@@ -30,8 +31,14 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Laravel\\Tinker\\Tests\\": "tests/"
-        }
+            "Laravel\\Tinker\\Tests\\": "tests/",
+            "App\\": "tests/fixtures/app",
+            "One\\Two\\": "tests/fixtures/vendor/one/two",
+            "Four\\Five\\": "tests/fixtures/vendor/four/five"
+        },
+        "files": [
+            "tests/helpers.php"
```

Comment:
Better to just duplicate the code in the the two places than bother with this tbh.

Reply:
Yeah I agree.

Reply:
Pushed a commit. Is that what you meant?

Reply:
No. Please delete the helpers file and replace the calls to the helpers with the code you previously had in the helpers.

Reply:
Code I previously had in the helpers? I'm not following sorry.

Reply:
No, it didn't? You've added them:

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/2829600/73689355-f5872d80-46c5-11ea-8ea4-9d647fef6ce7.png">

Reply:
Right, I don't understand what you're suggesting I do.

Reply:
Delete that file you have added and replace all calls to the methods you've added with just the actual code.

Reply:
It's a unit test though. There's no `config()` helper otherwise.